### PR TITLE
Describe how to convert existing clusters to use an RKE template

### PR DESCRIPTION
**Pull request by catherineluse**
This PR addresses this issue https://github.com/rancher/docs/issues/1966

### CODE REVIEW

File: content/rancher/v2.x/en/admin-settings/rke-templates/_index.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ Admins control which cluster options can be changed by end users. RKE templates
 
 If a cluster was created with an RKE template, you can't change it to a different RKE template. You can only update the cluster to a new revision of the same template.
 
-To summarize, RKE templates allow DevOps and security teams to:
+As of Rancher v2.3.3, you can [save the configuration of an existing cluster as an RKE template.]({{<baseurl>}}/rancher/v2.x/en/admin-settings/rke-templates/applying-templates/#converting-an-existing-cluster-to-use-an-rke-template) Then the cluster's settings can only be changed if the template is updated.
+
```
Comment (prachidamle) [inline content/rancher/v2.x/en/admin-settings/rke-templates/_index.md]: Maybe also mention that the template created can also be used to launch new clusters.

File: content/rancher/v2.x/en/admin-settings/rke-templates/_index.md [hunk 2]
```diff
@@ -48,7 +50,10 @@ RKE templates are supported for Rancher-provisioned clusters. The templates can
  
 RKE templates are for defining Kubernetes and Rancher settings. Node templates are responsible for configuring nodes. For tips on how to use RKE templates in conjunction with hardware, refer to [RKE Templates and Hardware]({{<baseurl>}}/rancher/v2.x/en/admin-settings/rke-templates/rke-templates-and-hardware).
 
-RKE templates can be applied to new clusters, but not existing clusters.
+RKE templates can be applied to new clusters.
```
Comment (prachidamle) [inline content/rancher/v2.x/en/admin-settings/rke-templates/_index.md]: Maybe we should change this to say, "RKE templates can be created from scratch to pre-define cluster configuration and they applied to launch new clusters, or templates can also be exported from existing running clusters"

File: content/rancher/v2.x/en/admin-settings/rke-templates/_index.md [hunk 3]
```diff
@@ -91,9 +96,12 @@ You can [create a cluster from a template]({{<baseurl>}}/rancher/v2.x/en/admin-s
 
 If the RKE template owner creates a new revision of the template, you can [upgrade your cluster to that revision.]({{<baseurl>}}/rancher/v2.x/en/admin-settings/rke-templates/applying-templates/#updating-a-cluster-created-with-an-rke-template)
 
-RKE templates can only be applied to new clusters, not existing clusters.
+RKE templates can be applied to new clusters.
```
Comment (prachidamle) [inline content/rancher/v2.x/en/admin-settings/rke-templates/_index.md]: Same comment as before.

File: content/rancher/v2.x/en/admin-settings/rke-templates/creating-and-revising/_index.md [hunk 4]
```diff
@@ -7,10 +7,24 @@ This section describes how to manage RKE templates and revisions. You an create,
...
+- [Updating a template](#updating-a-template)
+- [Deleting a template](#deleting-a-template)
+- [Creating a revision based on the default revision](#creating-a-revision-based-on-the-default-revision)
+- [Creating a revision based on a cloned revision](#creating-a-revision-based-on-a-cloned-revision)
+- [Disabling a template revision](#disabling-a-template-revision)
+- [Re-enabling a disabled template revision](#re-enabling-a-disabled-template-revision)
+- [Setting a template revision as default](#setting-a-template-revision-as-default)
+- [Deleting a template revision](#deleting-a-template-revision)
+- [Upgrading a cluster to use a new template revision](#upgrading-a-cluster-to-use-a-new-template-revision)
+
```
Comment (prachidamle) [inline content/rancher/v2.x/en/admin-settings/rke-templates/creating-and-revising/_index.md]: Maybe add a new section here "Exporting a running cluster to a new RKE template and revision"

File: content/rancher/v2.x/en/admin-settings/rke-templates/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/admin-settings/rke-templates/_index.md]: added "The new template can also be used to launch new clusters."

File: content/rancher/v2.x/en/admin-settings/rke-templates/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/admin-settings/rke-templates/_index.md]: Updated

File: content/rancher/v2.x/en/admin-settings/rke-templates/_index.md (hunk 3 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/admin-settings/rke-templates/_index.md]: Updated

File: content/rancher/v2.x/en/admin-settings/rke-templates/creating-and-revising/_index.md (hunk 4 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/admin-settings/rke-templates/creating-and-revising/_index.md]: added section

**Comment by catherineluse** [comment]
I messed up the rebase, so I have put the same changes on this other branch that is based on staging. https://github.com/rancher/docs/pull/2032

**Pull request closed without merging**